### DISCUSSION TITLE
participantRoles metadata

### DISCUSSION THREAD

**Discussion by vimal-varghese** [scolladon/sfdx-git-delta#417] [Q&A]
Hello All,
Could someone please confirm if the **participantRoles** metadata is currently supported by sfdx-git-delta tool (version 5.8.0).

We recently came across a situation where our delta package was not able to get the **participantRoles** folder and the underlying content in our delta package.
After few research, we came to know that the participantRoles metadata falls under **Financial Services Cloud Metadata API**, unlike the other metadata from our org.

Reference:
https://developer.salesforce.com/docs/atlas.en-us.232.0.financial_services_cloud_object_reference.meta/financial_services_cloud_object_reference/fsc_meta_participantrole.htm

We also checked at the metadata file found under https://github.com/scolladon/sfdx-git-delta/tree/main/src/metadata and we could **not** find this particular metadata included.

Please let us know if there is a plan to include the above Financial Services Cloud Metadata in the future or Is there any workaround for such metadata to be included in the delta package during creation.

Thank you so much for making our life easy when it come to delta package creation with the SGD plugin. 
It is a great tool which we totally depend on and it saves lot of our valuable time.

**Comment by scolladon** [comment]
I checked and it is not included in our metadata list
If you want it we can it for the next release
